impress-org/givewp
# New: Fee recovery works

## Description

This PR resolves the issue in the Classic Donation template where fee recovery amounts were not displayed in Donation Summary section, nor updated upon donation amount change.

## Affects

Fee recovery - Donation Summary Section

## Visuals

![Screen Capture_select-area_20211202180221](https://user-images.githubusercontent.com/4222590/144468726-5cb96735-4ccf-406d-a2f7-78193b222425.gif)


## Testing Instructions

1. Install Fee Recovery add-on and enable Donation Summary
2. Try every option available for Fee Recovery ( forced opt-in, donor opt-in ).
3. Verify that fee recovery is displaying correct amounts in Donation Summary section. 

## Pre-review Checklist
